testing/testcafe/tests/navigation/treeView/common.ts

```diff
@@ -38,6 +38,88 @@ test('Treeview search, selectAll item and nodes should be focused in DOM element
   items: employees,
 }));
 
+test('Treeview items focus order should be correct when changing showCheckBoxesMode from normal to selectAll at runtime', async (t) => {
+  const treeView = new TreeView('#container');
+  const selectAllItem = treeView.getSelectAllItem();
+  const node = treeView.getNode(0);
+
+  await treeView.option('showCheckBoxesMode', 'selectAll');
+
+  await t.pressKey('tab')
+    .expect(selectAllItem.hasClass('dx-state-focused'))
```

Comment:
Let's extract it to model helper and `getSelectAllItem()` should return target component that will have `isFocused`